### PR TITLE
Fixed font stylings for the intro section on MozFest primary pages

### LINKED ISSUE #3207: MozFest: subpage banner styling
@mmmavis can you please make the banner on all the mozfest sub pages look and resize the same as on the main sites Primary pages?

Desktop screen cap from main site:
image

Desktop screen cap from mozfest: looks good except the font style is different
image

Medium screen cap from main site:
image

Medium screen cap from mozfest: cut out and banner size is odd
image

Narrow screen cap from main site:
image

Narrow screen cap from mozfest: cut out is missing and banner size is odd
image

cc @sabrinang

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3207 

Screenshot_2019-05-22 Mozfest - Spaces page title en

### CODE REVIEW

File: network-api/networkapi/mozfest/templates/partials/primary_heroguts.html [hunk 1]
```diff
@@ -25,9 +25,9 @@ <h1 class="h2-heading mb-0">{% if page.header %}{{ page.header }}{% else %}{{ pa
   <div class="container">
     <div class="row no-gutters">
       <div class="cutout pt-4 col-12 {% if homepage != True %}col-md-8{% endif %}">
-        <h1 class="h2-heading mb-0">{% if root.title %}{{ root.title }}{% elif page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}</h1>
+        <h1 class="h1-heading mb-0">{% if root.title %}{{ root.title }}{% elif page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}</h1>
         {% if page.intro %}
-        <p class="body-large mt-3 mb-0">{{ page.intro | richtext }}</p>
+        <div class="intro body-large mt-3">{{ page.intro | richtext }}</div>
```
Comment (mmmavis) [inline network-api/networkapi/mozfest/templates/partials/primary_heroguts.html]: `{{ page.intro | richtext }}` returns block-level markup (e.g., `<p>`, `<ul>` ) and hence can't be wrapped within  the `<p>` wrapper

**Comment by mmmavis** [comment]
Review apps seem to be not working properly today. @sabrinang if you wanna test this out you can do it from my laptop / local build.

File: network-api/networkapi/mozfest/templates/partials/primary_heroguts.html (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/mozfest/templates/partials/primary_heroguts.html]: oh dang, good catch!

**Comment by Pomax** [comment]
@mavis if you manually rebase the review app should kick in properly again.

**Pull request merged**